### ISSUE TITLE
FusedBatchNorm not supported yet !

### ISSUE THREAD

**Issue by iChiaGuo** [Tencent/ncnn#490]
FusedBatchNorm not supported yet !
这个层如何自定义？

**Comment by lzx1413** [comment]
可参考wiki

**Comment by iChiaGuo** [comment]
@lzx1413 他的实现和batchnorm有什么区别，我看[这里](https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/fused-batch-norm)和[这个](https://stackoverflow.com/questions/43999951/difference-between-tf-nn-batch-normalization-and-tf-nn-fused-batch-norm)，好像没有什么区别，我用batchnorm来代替加载模型出错

**Comment by nihui** [comment]
duplicate of https://github.com/Tencent/ncnn/issues/5
